static/app/utils/profiling/gl/utils.spec.tsx

```diff
@@ -182,7 +182,7 @@ describe('createShader', () => {
       shaderSource: jest.fn(),
       compileShader: jest.fn(),
       getShaderParameter: jest.fn().mockImplementation(() => 1),
-      COMPILE_STATUS: 1,
+      COMPILE_STATUS: 1 as any,
```

Comment:
the type here is like `3142 | undefined` and its a test so i'm just casting it to any cc @JonasBa